### PR TITLE
Several updates to MAINTAINERS file

### PR DESCRIPTION
**Pull request by icecrime**
- Move docs approval section
- Sort names alphabetically
- Rename status labels to stay in sync with repo
- Add a new maintainer category entirely composed of @thaJeztah (:tada:)

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -600,6 +616,11 @@ made through a pull request.
 	Email = "SvenDowideit@home.org.au"
 	GitHub = "SvenDowideit"
 
+    [people.thajeztah]
```
Comment (jfrazelle) [inline MAINTAINERS]: tabs hehehehehhe

Comment (jfrazelle) [inline MAINTAINERS]: relevant https://github.com/jfrazelle/.vim/blob/master/vimrc.txt#L234

**Comment by jfrazelle** [comment]
but other than LGTM yay curators

File: MAINTAINERS (hunk 1 above)
Comment (icecrime) [inline MAINTAINERS]: No way, I was *certain* to get them all right...

**Comment by moxiegirl** [comment]
LGTM if we can call the category "thaJetzah"

**Comment by jfrazelle** [comment]
lol hahah

File: MAINTAINERS [hunk 2]
```diff
@@ -600,6 +616,11 @@ made through a pull request.
 	Email = "SvenDowideit@home.org.au"
 	GitHub = "SvenDowideit"
 
+	[people.thajeztah]
+	Name = "Sebastiaan van Stijn"
+	Email = "sebastiaan@gone.nl"
```
Comment (thaJeztah) [inline MAINTAINERS]: Should this match the email I'm using for signing-off? In that case `github@gone.nl` :)

**Comment by thaJeztah** [comment]
> LGTM if we can call the category "thaJetzah"

LOLz.


One small nit, apart from that, LGTM

Thanks all! ❤️

File: MAINTAINERS [hunk 3]
```diff
@@ -600,6 +616,11 @@ made through a pull request.
 	Email = "SvenDowideit@home.org.au"
 	GitHub = "SvenDowideit"
 
+	[people.thajeztah]
```
Comment (thaJeztah) [inline MAINTAINERS]: Oh, is this case-sensitive? `s/thajeztah/thaJeztah/`

Comment (icecrime) [inline MAINTAINERS]: I think the section are all lowercased.

File: MAINTAINERS (hunk 2 above)
Comment (icecrime) [inline MAINTAINERS]: It doesn't have to match, so just let me know which one you'd rather use :-)

File: MAINTAINERS (hunk 3 above)
Comment (thaJeztah) [inline MAINTAINERS]: Then, probably line 492 should be lowercased.

File: MAINTAINERS (hunk 2 above)
Comment (thaJeztah) [inline MAINTAINERS]: I think github@gone.nl would be better, just to separate things.

(Might change it to `@dockr.nl` one day, which I registered for fun :smile:)

**Comment by icecrime** [comment]
Updated!

**Comment by thaJeztah** [comment]
LGTM :smiley:

**Comment by thaJeztah** [comment]
Yay! 💃

**Comment by crosbymichael** [comment]
@thaJeztah 

power

**Comment by ahmetalpbalkan** [comment]
congrats @thaJeztah

**Comment by thaJeztah** [comment]
I'm fully aware of that! Will give my 100% to try and live up to that!

Thanks, @ahmetalpbalkan !

**Pull request merged**